aspnetcore/data/ef-rp/sort-filter-page.md

```diff
@@ -254,6 +254,8 @@ In the *SchoolViewModels* folder, add a *EnrollmentDateGroup.cs* with the follow
 
 ### Update the About page model
 
+The web templates in ASP.NET Core 2.2 do not include the About page. If you are using ASP.NET Core 2.2, create the About Razor Page.
```

Comment:
Can we expect them at this point in this tutorial to know how to create a Razor page?